### ISSUE TITLE
`tau trial export` should not attempt to export zero size data

### ISSUE THREAD

**Issue by zbeekman** [ParaToolsInc/taucmdr#96]
```
$ tau trial export
edu.uoregon.tau.perfdmf.DataSourceException: profiles type: no profiles specified
        at edu.uoregon.tau.perfdmf.UtilFncs.initializeDataSource(UtilFncs.java:546)
        at edu.uoregon.tau.perfdmf.UtilFncs.initializeDataSource(UtilFncs.java:513)
        at edu.uoregon.tau.paraprof.ParaProf.main(ParaProf.java:663)
```

See ${my_home_dir}/CRAFT/.tau

**Comment by zbeekman** [comment]
hmmm on second thought, this looks like it may be due to the fact that pprof/paraprof need java?

**Comment by zbeekman** [comment]
OK.... tried loading java, but then I realized that TAU CMDR never created profile data from the run... something is wrong here. This particular issue seems to be due to the fact that there is no profile data, so I'll close it.

**Comment by jlinford** [comment]
`tau trial export` shouldn't try to export trials with data size <= 0.

**Comment by khsa1** [comment]
Fixed in 0e931b5.
